gpflow/models/gpr.py

```diff
@@ -81,14 +83,14 @@ def log_marginal_likelihood(self) -> tf.Tensor:
             \log p(Y | \theta).
 
         """
-        X, Y = self.data
-        K = self.kernel(X)
+        X_data, Y_data = self.data
```

Comment:
Why has this method been changed?

Reply:
No major reason. Just renaming to make variable names the same

The classes in posterior.py use labelling along the lines of 'X_data' and 'Y_data', so changed all references within the gpr.py from  'X'/'Y' to 'X_data'/'Y_data' so it's the same 

(happy to change back if you want though)